public/app/features/explore/ExploreDrawer.tsx

```diff
@@ -43,7 +43,7 @@ const getStyles = stylesFactory((theme: GrafanaTheme2) => {
       left: calc(50% - 100px) !important;
       top: -4px !important;
       cursor: grab;
-      border-radius: 4px;
+      border-radius: ${theme.shape.borderRadius(2)};
```

Comment:
In this case we can use the `theme.shape.radius.pill` 😄

Reply:
Just checked whether `theme.shape.radius.pill` changes it drastically and it doesn't. So I changed it.
Before:
<img width="409" alt="Screenshot 2023-03-29 at 14 50 33" src="https://user-images.githubusercontent.com/48948963/228542470-c2f23852-2d25-450f-8989-5c52ee319e7c.png">
After:
<img width="336" alt="Screenshot 2023-03-29 at 14 57 38" src="https://user-images.githubusercontent.com/48948963/228542599-359dc292-ba0b-4006-85b7-0e27443942eb.png">